src/options/opt.cc

```diff
@@ -243,8 +243,10 @@ LOCAL_NODISCARD(Ret fix_mutopt(const conopt_t& glob,
     if (is_default.cond_goto_param)  real.cond_goto_param  = real.api_sigil;
     if (is_default.state_set_param)  real.state_set_param  = real.api_sigil;
     if (is_default.tags_expression)  real.tags_expression  = real.api_sigil;
+
+    bool is_c_type_lang = (glob.lang == Lang::C || glob.lang == Lang::D);
```

Comment:
Perhaps `need_semi` is a better name? And move declaration inside of if as it's needed there.